README.md

```diff
@@ -70,4 +70,4 @@ bundle exec cap staging deploy:rake task=db:seed
 
 ## Contributing
 
-The contribution guidelines can be found [here](doc/contributing.md).
\ No newline at end of file
+The contribution guidelines can be found [here](doc/contributing.md).
```

Comment:
These two lines look identical :o

Reply:
A [newline](http://pubs.opengroup.org/onlinepubs/9699919799/basedefs/V1_chap03.html#tag_03_206) at the end of the file has been added.